### PR TITLE
add doc: Debug With Sniffer

### PR DESCRIPTION
**Pull request by dpezely**
new doc: Confirm router runtime behavior using a virtual LoRaWAN device with a software-based protocol sniffer.

This how-to doc will have better reach being on the Docs website than living only with a code repo, but we'll link to this from relevant repos such as [router](https://github.com/helium/router).

These instructions have been reviewed by router folks and @lthiery who created the various tools mentioned.

EDIT: 
- Rendered version: https://deploy-preview-794--helium-docs.netlify.app/use-the-network/run-a-network-server
- same URL seems to get updated with each commit
- start with new bullet point after "run Console" and "deploy Console".

### CODE REVIEW

File: docs/use-the-network/run-a-network-server/debug-with-sniffer.mdx
Comment (lthiery) [inline docs/use-the-network/run-a-network-server/debug-with-sniffer.mdx:37]: all 1s?

File: docs/use-the-network/run-a-network-server/debug-with-sniffer.mdx
Comment (lthiery) [inline docs/use-the-network/run-a-network-server/debug-with-sniffer.mdx:56]: perhaps code block might make this render nicely?

File: docs/use-the-network/run-a-network-server/debug-with-sniffer.mdx
Comment (lthiery) [inline docs/use-the-network/run-a-network-server/debug-with-sniffer.mdx:183]: numbering

**Review by lthiery — COMMENTED** [review]
I didn't tag each one of them, but a few sections would probably render nicer if you put a code block in and some of the lists are all numbered 1.

File: docs/use-the-network/run-a-network-server/debug-with-sniffer.mdx
Comment (dpezely) [reply docs/use-the-network/run-a-network-server/debug-with-sniffer.mdx:37]: Fixed in commit be584ee.

FYI:

Markdown doesn't care about the actual number used, and each list gets rendered correctly:
https://deploy-preview-794--helium-docs.netlify.app/use-the-network/debug-with-sniffer/#overview

Probably only doc maintainers will ever see inside the .mdx file, so this is a technique for keeping future diffs cleaner, making future PRs easier/faster to review.

File: docs/use-the-network/run-a-network-server/debug-with-sniffer.mdx
Comment (dpezely) [reply docs/use-the-network/run-a-network-server/debug-with-sniffer.mdx:56]: Fixed in commit d5bb98b

Nice catch!  I didn't check if there's explicit styling for `bash` blocks that the previous indented block wouldn't get.

File: docs/use-the-network/run-a-network-server/debug-with-sniffer.mdx
Comment (dpezely) [reply docs/use-the-network/run-a-network-server/debug-with-sniffer.mdx:183]: Fixed in be584ee

**Comment by dpezely** [comment]
@kent-williams Here's some context on this PR:

The instructions within this document can be used by person technical enough to run or deploy Console + Router and not necessarily be restricted to just programmers.  It benefits new staff, partners and community contributors alike.

An earlier version of this document exists within Router's `docs/` subdirectory, so the content has been vetted there.  For benefit to the wider community, it makes sense having this "how to" live on the docs website.  

Once merged into the docs website, we'll update Router's README to point there and remove the earlier doc so that there's only one.

**Review by kent-williams — APPROVED** [review]
@dpezely hope you don't mind that I jumped in to re-org the directories just a bit and add the two newest docs to the sidenav. Otherwise it looks good to me, feel free to squash and merge at your leisure.

**Comment by dpezely** [comment]
Thank you, @kent-williams !

**Pull request merged**
